packages/data-models/src/options/options.js

```diff
@@ -137,4 +137,51 @@ export default class Options {
     }
     return parsed
   }
+
+  /**
+   * Converts optionItems to the object: { name of the option: currentValue }
+   *
+   * @returns {object}
+   */
+  get formatLabelValueList () {
+    const result = {}
```

Comment:
I think it should be `let` according to what we agreed upon previously

Reply:
Oh, I forgot to add es-lint exception, thank you for the catch - will fix